### DISCUSSION TITLE
Sir can I join your team

### DISCUSSION THREAD

**Discussion by onkarsinghsahil** [ROM-builders/temporary#11228] [Q&A]
Arrow successful build 
https://cirrus-ci.com/build/6206495514689536

Local manifest: https://github.com/onkarsinghsahil/local_manifest/blob/arrow/local_manifest.xml

Device tree: 
https://github.com/onkarsinghsahil/device_xiaomi_mido
Branch: wip (default)

Vendor tree:
https://github.com/onkarsinghsahil/proprietary_vendor_xiaomi

Kernal tree: 
https://github.com/onkarsinghsahil/kernel_mido

Telegram I'd: @error7951

**Comment by Apon77** [comment]
id mismatch
